Client/Frontend/ContextualHint/ContextualHintCopyProvider.swift

```diff
@@ -96,4 +106,16 @@ struct ContextualHintCopyProvider: FeatureFlaggable {
         default: return ""
         }
     }
+
+    private func getShoppingCopy(_ copyType: ContextualHintCopyType) -> String {
+        let hasOptedIn = prefs.boolForKey(PrefsKeys.Shopping2023OptIn) ?? false
+        var copy: String
+        switch copyType {
+        case .action:
+            copy = if hasOptedIn { CFRStrings.Shopping.OptedInAction } else { CFRStrings.Shopping.NotOptedInAction }
```

Comment:
I am of the opinion that this is not very idiomatic swift. The new inline ifs are great for multiple conditions, but, for single conditions, the ternary should be used as it's more straightforward. That's just my opinion, and I may just be doing an "old man yelling at could" here, so I'm curious what the rest of the team thinks. I'mma ask this in the iOS eng channel. This comment isn't blocking the PR unless @thatswinnie thinks it should

Reply:
I think should embrace new features of Swift, the  "?:" but it's not much better, other that the fact it's well known in the programming community.Funny thing, I have always struggle in the begging to put them in the corect order. For multiple conditions there's actually a separate `switch` implementation.

My only concert for the moment is that I'm not sure if we have Xcode 15 on Bitrise and this will not compile.

Reply:
I am still using Xcode 14 and this PR is not building. As far as I know Bitrise is still on Xcode 14 as well so the PR will need to be updated to run on Xcode 14. Sorry to spoil your fun with the new language features.